src/components/Select/Select.mdx

```diff
@@ -151,7 +151,8 @@ export function SelectControlledExample() {
 If your application uses uncontrolled components, you can pass a React `ref`
 prop to the DS Select component to get the selected value from the DOM. Note
 that this example uses a `Form` and a `Button` to submit the form, only then
-will the value be available.
+will the value be available. Additionally, you may pass a `defaultValue` prop
```

Comment:
Just add a note that it's mostly for the uncontrolled flow.

Reply:
This is under the "Uncontrolled Component" section. Do you still think I should add the note?

Reply:
No! I did not expand the view in github so missed the heading. Thanks for letting me know and keep it as is.